client/ayon_core/modules/deadline/plugins/publish/collect_pools.py

```diff
@@ -41,11 +41,7 @@ class CollectDeadlinePools(pyblish.api.InstancePlugin,
                 "renderlayer",
                 "maxrender",
                 "usdrender",
-                "redshift_rop",
-                "arnold_rop",
-                "mantra_rop",
-                "karma_rop",
-                "vray_rop",
+                "render.farm.hou",
```

Comment:
I'm pretty sure that by doing this you are losing the Pool selection attribute defs. As such, this seems wrong?

Reply:
correct! 
Since you have mentioned it. 
This problem also occurs with cache families that depend on `"publish.hou"` and it doesn't happen with `collect_pools` only but also `submit_houdini_render_deadline` and `submit_houdini_cache_deadline` 

The thing is How to add the families to these plugins without making them run `process`.. 
e.g `submit_houdini_render_deadline` will submit a job to deadline where I only want the `get_attribute_defs`. 

Should we move these `get_attribute_defs` to a dedicated collector ? 
It can be another issue (refactor Deadline submission) which is an issue that _Hopfully_ I'm going/be able to tackle soon 

| family add to bools | removed from bools |
|--|--|
| ![image](https://github.com/ynput/ayon-core/assets/20871534/2a062d44-a695-4a68-a495-12d97bc411fc) | ![image](https://github.com/ynput/ayon-core/assets/20871534/867b6368-b0be-472b-8922-6a5c69871c19)  |

Reply:
This targeting of families to show specific attributes - I believe @iLLiCiTiT had some opinions on that. So probably best to check with him how he'd recommend approaching this.

Reply:
I came up with a solution. I removed the intermediate family `render.farm.hou` it was unnecessary anyways.   
Now, it should work normally.
![image](https://github.com/ynput/ayon-core/assets/20871534/925a0ab7-1f7e-430a-9351-812074e11472)